IPython/core/debugger.py

```diff
@@ -2,18 +2,88 @@
 """
 Pdb debugger class.
 
+
+This is an extension to PDB which adds a number of new features.
+Note that there is also the `IPython.terminal.debugger` class which provides UI
+improvements.
+
+We also strongly recommend to use this via the `ipdb` package, which provides
+extra configuration options.
+
+Amoung other this subclass of PDB expose here:
+ - support many IPython magics like pdef/psource.
+ - allow to hide some frames in tracebacks
+ - allow to skip some frames.
```

Comment:
This currently looks a bit confusing IMO.

```suggestion
Among other things, this subclass of PDB:
 - supports many IPython magics like pdef/psource
 - hide frames in tracebacks based on `__tracebackhide__`
 - allows to skip frames based on `__debuggerskip__`
```